exercises/word-count/example.rs

```diff
@@ -1,3 +1,5 @@
+use std::convert::AsRef;
```

Comment:
as luck would have it, this is not a warning. word_count has a legitimate use of `as_ref`. Doesn't matter, I've seen what I need to see.